src/features/courses/course-controller.ts

```diff
@@ -302,6 +302,10 @@ class CourseController {
             where[`$enrolledStudents.${StudentEnrollment.rawAttributes.userId.field}$`] = options.filter.enrolledUserId;
         }
 
+        if (!_.has(where, 'active')) {
```

Comment:
If there's no `active` flag, we force only active courses? What about if we want both active and inactive courses?

Reply:
our active flag is soft delete, we should never be fetching inactive anything
technically old courses will remain active